src/QuickMarcEditor/QuickMarcEditorRows/FixedField/BibliographicFixedField.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  BytesField,
+} from '../BytesField';
+
+const BibliographicFixedField = ({ name, config }) => {
```

Comment:
I think with the new implementation this separation between `BibliographicFixedField`, `AuthorityFixedField` and `HoldingsFixedField` components is not needed - they have the same api and the field configuration is coming from back-end so we can have just one `FixedField` component